air-script/src/cli/transpile.rs

```diff
@@ -1,9 +1,12 @@
-use std::{fs, path::PathBuf};
+use std::{fs, path::PathBuf, sync::Arc};
+
+use miden_diagnostics::term::termcolor::ColorChoice;
+use miden_diagnostics::{CodeMap, DefaultEmitter, DiagnosticsHandler};
 use structopt::StructOpt;
 
 use codegen_winter::CodeGenerator;
 use ir::AirIR;
-use parser::parse;
+use parser::{ast::Source, Parser};
```

Comment:
I'm a fan of breaking up imports rather than clumping them together like that, as it makes it harder to see at a glance where things are coming from. Similarly for why I break things up into three groups - items from the standard library, followed by items from third-party crates, followed by items from the current crate. That said, I'm happy to follow whatever the team preference is, though I would be curious to know what motivates that particular choice.

Reply:
Not a strong preference - it's just how we've been doing it in all other places. If you'd prefer to keep the 3 groups separate that's fine, but I would consolidate imports from the same crate.